src/libraries/System.Private.Xml/tests/XmlWriter/XmlTextWriterTests.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace System.Xml.Tests
```

Comment:
These tests no longer run on .NET Framework any more. I wonder whether if you added the TFM temporarily into \src\libraries\System.private.xml\tests\XmlWriter\System.Xml.RW.XmlWriter.Tests.csproj it would do so, and validate your assertions here.

Reply:
See https://github.com/dotnet/runtime/pull/74787#issuecomment-1231902409.